### PR TITLE
[Ukrainian] Translate `/plugins/console-time.md`

### PR DESCRIPTION
**Pull request by LWJerri**

### CODE REVIEW

File: site/docs/uk/plugins/console-time.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Виведення логів у консоль під час налагодження
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:1]:
```suggestion
# Вимірювання часу обробки у консоль під час налагодження
```

File: site/docs/uk/plugins/console-time.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Якщо ви знайомі з JavaScript/TypeScript, ви, напевно, використовували [`console.log`](https://developer.mozilla.org/en-US/docs/Web/API/Console/log) або [`console.time`](https://developer.mozilla.org/en-US/docs/Web/API/Console/time), щоб перевірити, що відбувається під час налагодження.
+Під час роботи з вашим ботом або middlewares ви, можливо, захочете перевірити щось подібне: Що відбулося і як довго це зайняло?
+
+Цей плагін призначений для відлагодження окремих запитів з метою вирішення окремих проблем.
+Але у продакшен середовищі, ймовірно, вам знадобиться щось протилежне, щоб отримати загальну інформацію.
+Наприклад, під час відлагодження проблеми з `/start` ви перевіряєте окреме оновлення Telegram.
+В контексті продакшина вас цікавлять всі повідомлення `/start`, що відбуваються.
+Ця бібліотека призначена для допомоги в роботі з окремими оновленнями.
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:10]:
```suggestion
Цей плагін призначений для відлагодження окремих запитів з метою вирішення окремих проблем.
У продакшен середовищі вам знадобиться щось інше, щоб отримати загальну інформацію про всі запити.
Наприклад, під час відлагодження проблеми з командою `/start` ви перевіряєте окреме оновлення Telegram.
В контексті продакшену вас скоріше цікавлять взагалі всі оновлення з командою `/start`, але ця бібліотека призначена для допомоги в роботі з окремими оновленнями.
```

File: site/docs/uk/plugins/console-time.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+```ts
+import { generateUpdateMiddleware } from "telegraf-middleware-console-time";
+
+if (process.env.NODE_ENV !== "production") {
+  bot.use(generateUpdateMiddleware());
+}
+
+// Ваша реалізація
+bot.command("старт" /* , ... */);
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:22]:
```suggestion
bot.command("start" /* , ... */);
```

File: site/docs/uk/plugins/console-time.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+які будуть виводи із такого тексту:
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:25]:
```suggestion
Це код виведе щось на зразок цього:
```

File: site/docs/uk/plugins/console-time.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+```plaintext
+2020-03-31T14:32:36.974Z 490af message text Andrey 6 /старт: 926.247ms
+2020-03-31T14:32:57.750Z 490ag message text Andrey 6 /старт: 914.764ms
+2020-03-31T14:33:01.188Z 490ah message text Andrey 5 /стоп: 302.666ms
+2020-03-31T14:46:11.385Z 490ai message text Andrey 6 /старт: 892.452ms
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:31]:
```suggestion
2020-03-31T14:32:36.974Z 490af message text Andrey 6 /start: 926.247ms
2020-03-31T14:32:57.750Z 490ag message text Andrey 6 /start: 914.764ms
2020-03-31T14:33:01.188Z 490ah message text Andrey 5 /stop: 302.666ms
2020-03-31T14:46:11.385Z 490ai message text Andrey 6 /start: 892.452ms
```

File: site/docs/uk/plugins/console-time.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+`490af` - це `update_id`.
+
+Число перед командами - це загальна довжина вмісту.
+Це корисно при розгляді максимальної довжини для таких речей, як дані зворотного виклику.
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:37]:
```suggestion
Це корисно, коли потрібно врахувати максимальну довжину для таких речей, як дані зворотного виклику.
```

File: site/docs/uk/plugins/console-time.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+Сам вміст скорочується, щоб запобігти засміченню журналу.
+
+## Відлагодження вашого middleware
+
+Коли ви створюєте власний middleware або вважаєте, що інший middleware повільний, ви можете використовувати ці middlewares, щоб створити профіль часу виконання.
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:43]:
```suggestion
Коли ви створюєте власний middleware або вважаєте, що якийсь middleware повільний, ви можете використовувати ці middleware, щоб створити профіль часу виконання.
```

File: site/docs/uk/plugins/console-time.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+```ts
+import {
+  generateAfterMiddleware,
+  generateBeforeMiddleware,
+} from "telegraf-middleware-console-time";
+
+const bot = new Bot(/* ... */);
+
+// Використовуйте BeforeMiddleware перед завантаженням тестувального middleware.
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:53]:
```suggestion
// Використовуйте BeforeMiddleware перед встановленням middleware, що тестується.
```

File: site/docs/uk/plugins/console-time.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+bot.use(generateBeforeMiddleware("foo"));
+
+// Middleware, який потрібно протестувати.
+bot.use(); /* ... */
+
+// Використовуйте AfterMiddleware після завантаження middleware, який ви тестуєте з тією самою міткою.
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:59]:
```suggestion
// Використовуйте AfterMiddleware з тією самою міткою, після встановлення middleware, що тестується.
```

File: site/docs/uk/plugins/console-time.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+bot.use(generateAfterMiddleware("foo"));
+
+// Інші middlewares/реалізації, вони займатимуть "внутрішній" час при використанні.
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:62]:
```suggestion
// Усі інші middleware та код боту, вони рахуватимуться як "inner" при вимірюванні часу.
```

File: site/docs/uk/plugins/console-time.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+bot.use(); /* ... */
+bot.on("message" /* ... */);
+```
+
+Цей вивід буде виглядати так:
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:67]:
```suggestion
Це код виведе щось на зразок цього:
```

**Review by deptyped — CHANGES_REQUESTED** [review]

File: site/docs/uk/plugins/console-time.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+```plaintext
+490ai foo before: 304.185ms
+490ai foo inner: 83.122ms
+490ai foo after: 501.028ms
+490ai foo total: 891.849ms
+```
+
+Це свідчить про те, що перевірений middleware виконувався сам по собі 800мс і, можливо, його продуктивність не є достатньою.
+
+## Опис плагіна
```
Comment (deptyped) [inline site/docs/uk/plugins/console-time.md:78]:
```suggestion
## Загальні відомості про плагін
```

**Review by deptyped — CHANGES_REQUESTED** [review]

File: site/docs/uk/plugins/console-time.md (hunk 2 above)
Comment (Nazar-Ant) [reply site/docs/uk/plugins/console-time.md:10]: Чи варто нам паплюжити переклад словом "продакшн"?

```suggestion
Цей плагін призначений для відлагодження окремих запитів з метою вирішення окремих проблем.
У production середовищі вам знадобиться щось інше, щоб отримати загальну інформацію про всі запити.
Наприклад, під час відлагодження проблеми з командою `/start` ви перевіряєте окреме оновлення Telegram.
У контексті production вас скоріше цікавлять взагалі всі оновлення з командою `/start`, але ця бібліотека призначена для допомоги в роботі з окремими оновленнями.
```

File: site/docs/uk/plugins/console-time.md [hunk 13]
```diff
@@ -0,0 +1,80 @@
+# Виведення логів у консоль під час налагодження
+
+Якщо ви знайомі з JavaScript/TypeScript, ви, напевно, використовували [`console.log`](https://developer.mozilla.org/en-US/docs/Web/API/Console/log) або [`console.time`](https://developer.mozilla.org/en-US/docs/Web/API/Console/time), щоб перевірити, що відбувається під час налагодження.
+Під час роботи з вашим ботом або middlewares ви, можливо, захочете перевірити щось подібне: Що відбулося і як довго це зайняло?
+
+Цей плагін призначений для відлагодження окремих запитів з метою вирішення окремих проблем.
```
Comment (Nazar-Ant) [inline site/docs/uk/plugins/console-time.md:6]:
```suggestion
Цей плагін призначений для налагодження окремих запитів з метою вирішення окремих проблем.
```

File: site/docs/uk/plugins/console-time.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+Але у продакшен середовищі, ймовірно, вам знадобиться щось протилежне, щоб отримати загальну інформацію.
+Наприклад, під час відлагодження проблеми з `/start` ви перевіряєте окреме оновлення Telegram.
```
Comment (Nazar-Ant) [inline site/docs/uk/plugins/console-time.md:8]:
```suggestion
Наприклад, під час налагодження проблеми з `/start` ви перевіряєте окреме оновлення Telegram.
```

File: site/docs/uk/plugins/console-time.md [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+`490af` - це `update_id`.
+
+Число перед командами - це загальна довжина вмісту.
+Це корисно при розгляді максимальної довжини для таких речей, як дані зворотного виклику.
+
+Сам вміст скорочується, щоб запобігти засміченню журналу.
+
+## Відлагодження вашого middleware
```
Comment (Nazar-Ant) [inline site/docs/uk/plugins/console-time.md:41]:
```suggestion
## Налагодження вашого middleware
```

File: site/docs/uk/plugins/console-time.md [hunk 16]
```diff
@@ -0,0 +1,80 @@
...
+Цей вивід буде виглядати так:
+
+```plaintext
+490ai foo before: 304.185ms
+490ai foo inner: 83.122ms
+490ai foo after: 501.028ms
+490ai foo total: 891.849ms
+```
+
+Це свідчить про те, що перевірений middleware виконувався сам по собі 800мс і, можливо, його продуктивність не є достатньою.
```
Comment (Nazar-Ant) [inline site/docs/uk/plugins/console-time.md:76]:
```suggestion
Це свідчить про те, що перевірений middleware виконувався сам по собі 800 мс і, можливо, його продуктивність достатня.
```

**Review by Nazar-Ant — CHANGES_REQUESTED** [review]
...

File: site/docs/uk/plugins/console-time.md (hunk 10 above)
Comment (LWJerri) [reply site/docs/uk/plugins/console-time.md:61]: Why do you not translate the word "inner"?

File: site/docs/uk/plugins/console-time.md (hunk 16 above)
Comment (LWJerri) [reply site/docs/uk/plugins/console-time.md:75]: Changing one word for another has not changed anything, the meaning is the same.

**Comment by KnorpelSenf** [comment]
@LWJerri if you want to accept many suggestions, consider switching to the files tab
image
at https://github.com/grammyjs/website/pull/680/files and add many suggestions to a batch. That way, you can perform a single commit that collects them all.

File: site/docs/uk/plugins/console-time.md (hunk 16 above)
Comment (Nazar-Ant) [reply site/docs/uk/plugins/console-time.md:75]: По суті так, але тепер це легше читається і правильніше з точки зору мови, як на мене.

**Comment by LWJerri** [comment]
> @LWJerri if you want to accept many suggestions, consider switching to the files tab image at https://github.com/grammyjs/website/pull/680/files and add many suggestions to a batch. That way, you can perform a single commit that collects them all.

Yes, I'm aware of that, but I'm more comfortable doing reviews from this page.

**Review by Nazar-Ant — APPROVED** [review]

File: site/docs/uk/plugins/console-time.md (hunk 10 above)
Comment (deptyped) [reply site/docs/uk/plugins/console-time.md:61]: The measurement result does not contain "внутрішній". How is the user supposed to know it's about "inner"?

**Review by Nazar-Ant — APPROVED** [review]

**Pull request merged**